packages/hostScriptsServices/hostScripts/recreate_dappnode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Host script service executed with a timer to ensure dappmanager container is running, otherwise recreate dappnode to latest version
+# The edge case where the dappmanager container does not exist is not covered
+
+# Name of the dappmanager Docker container
+DAPPMANAGER_CONTAINER_NAME="DAppNodeCore-dappmanager.dnp.dappnode.eth"
+
+# Check the status of the container
+# Possible docker container status: created, exited, dead, created, restarting, paused and running
+STATUS=$(docker inspect --format='{{.State.Status}}' "$DAPPMANAGER_CONTAINER_NAME" 2>/dev/null)
+
+# For safety check for docker container status not expected instead of checking for running
```

Comment:
Why wouldn't `!= running` work? I do not see a case where this makes sense, except for the one in which you have different behavior for each of them

Reply:
You are right that would work and would be perfectly fine, but I am afraid of the behaviour of this script, specially in unexpected bug loops. I rather prefer to check for those positives instead of negatives.